easybuild/easyblocks/t/tensorflow.py

```diff
@@ -256,6 +256,9 @@ def configure_step(self):
                        r"\1, '--output_base=%s', '--install_base=%s'" % (tmpdir, os.path.join(tmpdir, 'inst_base')))]
         apply_regex_substitutions('configure.py', regex_subs)
 
+        # Tell Bazel to not use $HOME/.cache/bazel at all
+        # See site/docs/output_directories.md
```

Comment:
Please point to https://docs.bazel.build/versions/master/output_directories.html